it seems a bug that an array is indexed twice.

wykkyle opened the pull request:
‘paddle.gather’ has been applyed on array 'boxes' in the above code. 
(which is named as fg_inds before transform from tensor to numpy.array)
so, i think it should loop with sequnce index here.

wykkyle committed:
it seems a bug that an array is indexed twice.

‘paddle.gather’ has been applyed on array 'boxes' in the above code. 
(which is named as fg_inds before transform from tensor to numpy.array)
so, i think it should loop with sequnce index here.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/PaddlePaddle/PaddleDetection?pullRequest=5742) <br/>All committers have signed the CLA.

wykkyle committed:
array indexed twice by same index

it seems a bug that an array is indexed twice.

originally, array 'new_segm/gt_segm_per_im' and 'boxes/fg_rois' are indexed twice by the same index 'fg_inds_new/fg_inds'.
the first indexing takes places in the form of 'paddle.gather', the second indexing is explicitlly implemented in the loop.
delete the second indexing.

originally, i think, it does not report a bug  because 'fg_inds' happens to be [0,1,2,3,4,...,n-1].
